build.gradle.kts

```diff
@@ -0,0 +1,50 @@
+import configurations.Languages.attachLocalRepositories
+
+import dependencies.Libraries.ANDROID_BUILD_TOOLS
+
+buildscript {
+    repositories {
+        mavenCentral()
+        google()
+    }
+
+    dependencies {
+        classpath(ANDROID_BUILD_TOOLS)
+    }
+}
+
+allprojects {
+    group = "org.worldcubeassociation.tnoodle"
+    version = "0.14.0"
+    
+    attachLocalRepositories()
+}
+
+val releasePrefix = "TNoodle-WCA"
+
+tasks.create<Copy>("generateOfficialRelease") {
+    description = "Generate an official WCA release artifact. THIS WILL RUN TESTS!"
+    group = "WCA"
+
+    val targetProject = "webscrambles"
+
+    dependsOn(getTasksByName("check", true))
```

Comment:
Could we remove this? I don't think this is appropriate to do when building the asset. I'd prefer that we explicitly run `./gradlew check` and `./gradlew generateOfficialRelease` in Travis. I think this is also nice change to make for developers who just want to build a real jar file locally without waiting a very long time for tests to run, rather than forcing them to learn about yet another gradle task to run (`shadowJarOfficial`).

(don't forget to update this task description (it mentions tests) and the README as well)

Reply:
I have decoupled the task structure accordingly